pandas/tests/tseries/test_ticks.py

```diff
@@ -0,0 +1,207 @@
+# -*- coding: utf-8 -*-
+"""
+Tests for offsets.Tick and subclasses
+"""
+from datetime import datetime, timedelta
+
+import pytest
+import numpy as np
+
+from pandas import Timedelta, Timestamp
+from pandas.tseries import offsets
+from pandas.tseries.offsets import Hour, Minute, Second, Milli, Micro, Nano
+
+
+# ---------------------------------------------------------------------
+# Test Helpers
+
+tick_classes = [Hour, Minute, Second, Milli, Micro, Nano]
+
+
+def assertEq(offset, base, expected):
```

Comment:
put this in a common.py and import here

don't use camelCase ever! and spell it out

``assert_offset_equal``

Reply:
> don't use camelCase ever! and spell it out

I agree, this is taken from an existing version in test_offsets.  I'll move and rename both.

(Need to get rid of a whole bunch of camelCase in offsets, but that's gonna have to wait until the non-cosmetic stuff gets through).